convert2rhel/cert.py

```diff
@@ -58,3 +60,33 @@ def install(self):
             loggerinst.critical("OSError({0}): {1}".format(err.errno, err.strerror))
 
         loggerinst.debug("Certificate copied to %s." % self._system_cert_dir)
+
+    def remove(self):
+        """Removes certificate (.pem), which was copied in install method"""
+        loggerinst.info("Removing copy of certificate from %s." % self._system_cert_dir)
+
+        try:
+            os.remove(os.path.join(self._system_cert_dir, self._get_cert_filename()))
```

Comment:
```suggestion
            os.remove(self._target_cert_path)
```

There's too many cert paths and dirs, so it's better to distinguish them to easily tell them apart.

```python
def __init__(self):
    self._source_cert_dir = os.path.join(utils.DATA_DIR, "rhel-certs")
    self._target_cert_dir = "/etc/pki/product-default/"
    self._cert_filename = self._get_cert_filename()
    self._source_cert_path = self._get_source_cert_path()
    self._target_cert_path = self._get_target_cert_path()

def _get_cert_filename(self):
    pem_filename = None
    for filename in os.listdir(self._source_cert_dir):
        if filename.endswith(".pem"):
            pem_filename = filename
            break
    if not pem_filename:
        loggerinst.critical(
            "Error: System certificate (.pem) not found in %s." % self._source_cert_dir
        )
    return pem_filename

def _get_source_cert_path(self):
    os.path.join(self._source_cert_dir, self._cert_filename)

def _get_target_cert_path(self):
    os.path.join(self._target_cert_dir, self._cert_filename)
```